src/Libraries/CoreNodeModelsWpf/NodeViewCustomizations/DoubleInput.cs

```diff
@@ -28,7 +28,7 @@ public void CustomizeView(DoubleInput nodeModel, NodeView nodeView)
 
             tb.DataContext = nodeModel;
 
-            tb.BindToProperty(new Binding("Value")
+            tb.BindToProperty(new Binding("NumericalValue")
             {
                 Mode = BindingMode.TwoWay,
                 Converter = new DoubleInputDisplay(),
```

Comment:
more thoughts - and why I implemented this validation on the model : theres a few more wrinkles, like the` DynamoTextBox` actually sending an updateModelValue command manually so that undo/redo works for final input on focus loss.

I think what should be done is to add a validation rule to the binding that DynamoText box uses, the converter is not really for validation, it expects to get valid input. When the Text box attempts to send this update command it will check its validation errors and send an error instead on the node model if it encounters a numerical validation error.

I will try this out next, it may be possible to get rid of the extra `NumericalValue` property this way...